examples/TestDriver/ios/Podfile

```diff
@@ -35,3 +35,11 @@ target "TestDriver" do
     inherit! :search_paths
   end
 end
+
+# Force react-native-heap to be built as a static framework
+pre_install do |installer|
+    pod = installer.pod_targets.find { |p| p.name == 'react-native-heap'}
+    def pod.static_framework?
+        true
+    end
+end
```

Comment:
If we're adding this, should we add `use_frameworks!` at the top, too?

Reply:
Added.  It shouldn't matter to much except as an example, so might as well make it as close to what a user might encounter as possible.